package.json

```diff
@@ -12,17 +12,17 @@
     "examples": "./examples"
   },
   "files": [
+    "README.md",
     "THIRD_PARTY_LICENSES.txt",
     "index.js",
     "index.d.ts",
     "lib/**/*.js",
     "src/**/*.ts",
-    "examples/*",
-    "doc/api/**"
+    "examples/**"
   ],
   "scripts": {
     "test": "mocha && tsc --project test/unit/types",
-    "docs": "jsdoc -c conf.json"
+    "docs": "typedoc"
```

Comment:
is doc done differently now?

Reply:
Yes, the doc is built using typedoc and comments from the declaration files.  I put some info about this into README-DEV.